tests/test_Admin.py

```diff
@@ -115,6 +115,8 @@ def test_create_topics_api():
         a.create_topics([NewTopic("mytopic", 3, 2,
                                   config=["fails", "because not a dict"])])
 
+    fs = a.create_topics([NewTopic("mytopicDefault")])
+
```

Comment:
Also add a integration test to integration_test.py to verify that the topic is indeed created.